pkg/controller/user/importbatch.go

```diff
@@ -74,7 +75,8 @@ func (c *Controller) HandleImportBatch() http.Handler {
 			} else if created {
 				newUsers = append(newUsers, &batchUser)
 				if err := c.emailer.SendNewUserInvitation(ctx, user.Email); err != nil {
```

Comment:
Previously, this was a non-blocking operation (calling the firebase API). However, now it's blocking. SMTP sending is happening inline with the request, and SMTP is blocking. We either need to so something primitive (like push this into a background go routine and hope it finishes before the instance spins down) or build a job processing queue.

Reply:
Done as a simple goroutine. I think the firebase operation is still a blocking call? These seem to be pretty quick

Reply:
Firebase was a blocking HTTP call, but it queued the email. The HTTP response didn't depend on SMTP.